parsec/class/parsec_rwlock.h

```diff
@@ -82,9 +82,9 @@ typedef struct parsec_atomic_rwlock_s {
 } parsec_atomic_rwlock_t;
 
 #if defined(PARSEC_DEBUG_PARANOID)
-#define PARSEC_RWLOCK_UNLOCKED { PARSEC_ATOMIC_UNLOCKED, PARSEC_ATOMIC_UNLOCKED, 0, 0 }
+#define PARSEC_RWLOCK_UNLOCKED PARSEC_ATOMIC_UNLOCKED, PARSEC_ATOMIC_UNLOCKED, 0, 0,
```

Comment:
This change seems to unintended. Didn't we just make the change to add the brackets to the `PARSEC_RWLOCK_UNLOCKED`?

Reply:
Fixed (and applied the same changes to the non-debug-paranoid case)